client/klay_client.go

```diff
@@ -237,6 +237,17 @@ func (ec *Client) TransactionReceipt(ctx context.Context, txHash common.Hash) (*
 	return r, err
 }
 
+// TransactionReceiptRpcOutput returns the receipt of a transaction by transaction hash as a rpc output.
+func (ec *Client) TransactionReceiptRpcOutput(ctx context.Context, txHash common.Hash) (r map[string]interface{}, err error) {
+	err = ec.c.CallContext(ctx, &r, "klay_getTransactionReceipt", txHash)
+	if err == nil {
```

Comment:
how about `if err == nil && r == nil`.

Reply:
> Is it possible to generate an error in klay `GetTransactionReceipt`? It seems to be redundant.

I think it is possible. : Look another APIs. All APIs have an exception code.

Reply:
> how about `if err == nil && r == nil`.

Good. I reflect. :)

Reply:
What I was saying is that it would be nice to return this error in the `func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, hash common.Hash) (map[string]interface{}, error)`, because it is added the same error in all the code that calls the `func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, hash common.Hash) (map[string]interface{}, error)`. Will not you reflect on the code?

Reply:
Thanks. @nogang :) I will try to refactor duplicated code, later.